README.md

```diff
@@ -57,11 +57,14 @@ END
 }
 ```
 
+Note: The example above requires `smart_proxy_remote_execution_ssh` Smart Proxy
+plugin.
+
 #### POST /tasks/$TASK_ID/cancel
 Tries to cancel a task.
 
 ```
-curl -X POST localhost:8008/tasks/dd5a8306-0e52-4f68-9e83-c7f51c9e95c3/cancel -d '' 2>/dev/null | jq
```

Comment:
Why we dont like `j1` nomore?

Reply:
It is not really essential to what one might want to accomplish with this and I didn't want to force my preferred tools on anyone